src/caret_analyze/record/frequency.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 Research Institute of Systems Planning, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Optional, Tuple
+
+from .column import ColumnValue
+from .interface import RecordsInterface
+from .record_factory import RecordFactory, RecordsFactory
+
+
+class Frequency:
+
+    def __init__(
+        self,
+        records: RecordsInterface,
+        target_column: Optional[str] = None
+    ) -> None:
+        self._target_column = target_column or records.columns[0]
+        self._target_series = [ts for ts in records.get_column_series(
```

Comment:
Since `get_column_series` loops `append` for each record, its performance isn't good.
It's better to iterate records instead of calling `get_column_series` (see: https://github.com/tier4/CARET_analyze/blob/main/src/caret_analyze/record/response_time.py#L753)

Reply:
Fixed: https://github.com/tier4/CARET_analyze/pull/116/commits/f1e07d731e9cc2f9e9b50807fe1f3f86ff02b394